cpp/include/cudf/strings/detail/utilities.hpp

```diff
@@ -27,6 +27,23 @@ namespace cudf {
 namespace strings {
 namespace detail {
 
+/**
+ * @brief Create an offsets column to be a child of a strings column
+ *
+ * This will return the properly typed column to be filled in by the caller
+ * given the number of bytes to address.
+ *
+ * @param chars_bytes Number of bytes for the chars in the strings column
+ * @param count Number of elements for the offsets column
```

Comment:
Does this add 1 to account for there being `num_rows + 1` offsets? Documenting that clearly would be helpful.

Reply:
I read "number of elements for the offsets column" to imply this is already +1'd and it is the number of offsets elements. I mention this because clarification would be useful either way this goes.